src/scripts/main.js

```diff
@@ -1 +1,41 @@
 'use strict';
+
+const addText = (number, isError) => `
```

Comment:
In fact, this variable `number ` does not store a number, but the name of the promise, rename it.